src/app/Dashboard/LayoutTemplateGroup.tsx

```diff
@@ -200,6 +203,14 @@ export const KebabCatalogTileBadge: React.FC<KebabCatalogTileBadgeProps> = ({ te
     [onTemplateDelete, template.name]
   );
 
+  const handleClearCards = React.useCallback(
```

Comment:
Also remove this handler?

Reply:
yah I noticed and removed it

Reply:
check the new push please @tthvo

Reply:
Sorry seems to be still there for me tho.

Reply:
* as shown on Github.

Reply:
See it now.